source/scenario/newmission3task4.vwf.yaml

```diff
@@ -44,7 +44,7 @@ properties:
   - loadToolbox:
     - rover
-    - source/scenario/blockly/mission3b.xml
+    - source/scenario/blockly/mission3d.xml
```

Comment:
In ideal world, we'd name the files consistently as MissionXTaskY (no "new", no "mission3x")...